migrations/1677867813114_drop-cmr-preferences.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined
```

Comment:
I'm not an expert on node-pg-migrate, so it's more of a question than a comment.  Is this line `export.shorthands = undefined` necessary?

Reply:
looking at the [documentation](https://github.com/salsita/node-pg-migrate/blob/master/docs/migrations.md), I don't believe it is, it's just part of their template. I experimented with removing it and making a migration and it can be dropped

Reply:
sweet. thanks for looking into it.